srtcore/core.cpp

```diff
@@ -3284,19 +3281,6 @@ SRTSOCKET srt::CUDT::makeMePeerOf(SRTSOCKET peergroup, SRT_GROUP_TYPE gtp, uint3
                   << gp->id());
     }
 
-    {
-        ScopedLock glock (*gp->exp_groupLock());
-        if (gp->closing())
-        {
-            HLOGC(gmlog.Debug, log << CONID() << "makeMePeerOf: group $" << gp->id() << " is being closed, can't process");
-        }
-
-        if (was_empty)
-        {
-            gp->syncWithSocket(s->core(), HSD_RESPONDER);
```

Comment:
To fix the issue, it should keep previous group-wise sequences even if the group is empty.